src/video-player.js

```diff
@@ -118,16 +118,26 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     this._initPlaylistWidget();
     this._initJumpButtons();
     this._setVideoJsListeners(ready);
-    this._sendAnalytics(this.playerOptions);
   }
 
-  _sendAnalytics(options) {
+  _sendInternalAnalytics(additionalOptions = {}) {
+    if (this._internalAnalyticsSent) {
+      return;
+    }
```

Comment:
It seems to me that if we remove this if statement we will get the usage collected each time the source is changed right? if so, and only that is required to make it work, with no risks, maybe we should supporting the analytics whenever the source is changing?

Reply:
actually thats question for Raz - init event should be send only once, otherwise it has totally different meaning, maybe we should rename it to `video_player_source` ?